tools/polymer/example/handwritten/nussinov.pluto.c

```diff
@@ -0,0 +1,109 @@
+// File name: main.c
```

Comment:
what is the purpose of this handwritten directory?

Reply:
this is a @kumasento question if you know the answer offhand. (right now this is a first pass to merge the repos while preserving history and getting it to build)

Reply:
Fair point, we can iterate on this later.